src/policies.rs

```diff
@@ -175,12 +176,46 @@ fn verify_commit_signatures<G: Git>(
     commits: &Vec<Commit<'_>>,
     fingerprints: &HashMap<String, Fingerprint>,
 ) -> Result<PolicyResult, Box<dyn Error>> {
+
+    //hashmap of VerificationCommits
```

Comment:
If you need to be clearer about the types in the hashmap, I think fill them in over the _s, rather than having a comment.